tests/test_subclass.py

```diff
@@ -12,10 +12,9 @@ class LabeledVector(Vector2):
     """Subclass of Vector2 that defines an additional attribute."""
     label: str
 
-    def __new__(cls, x, y, label):
-        self = super().__new__(cls, x, y)
+    def __init__(self, x, y, label):
+        Vector2.__init__(self, x, y)
```

Comment:
`super()`

Reply:
Those 2 don't seem equivalent; you can check, all tests in that file fail if you make it `super()`.

Reply:
Nevermind, I'm just bad at code